### PR TITLE
multiple cosmetic fixes

### PR DESCRIPTION
**Pull request by CvH**
pls don't look at the overall changes, single commits provide better overview
Would be nice if someone could have a second look about the changes.

- fixed copyright headers `Copyright (C) 2016 Team LibreELEC` -> `Copyright (C) 2016-present Team LibreELEC`
- fixed copyright headers urls `https://www.libreelec.tv` & `https://LibreELEC.tv` -> `https://libreelec.tv`
- fixed empty long and short descriptions
- removed double newlines
- removed false newlines at EOF
- removed white spaces from the end of line
- removed double usage of `PKG_USE_CMAKE`
- removed `PKG_IS_ADDON="no"` and changed (to follow the overall standard)
```
...

PKG_AUTORECONF="yes"
```
to
```
...
PKG_AUTORECONF="yes"
```
- changed 
```
...

PKG_USE_CMAKE="no"
PKG_AUTORECONF="yes"
```
to
```
...
PKG_AUTORECONF="yes"
PKG_USE_CMAKE="no"
```

### CODE REVIEW

File: packages/addons/addon-depends/system-tools-depends/vim/package.mk [hunk 1]
```diff
@@ -53,4 +53,4 @@ pre_configure_target() {
 
 make_target() {
   :
-}
+}
```
Comment (MilhouseVH) [inline packages/addons/addon-depends/system-tools-depends/vim/package.mk]: Can you restore the newline (which means the second hunk isn't necessary)

File: packages/emulation/libretro-picodrive/package.mk [hunk 2]
```diff
@@ -79,5 +78,4 @@ makeinstall_target() {
   mkdir -p $SYSROOT_PREFIX/usr/lib/cmake/$PKG_NAME
   cp $PKG_LIBPATH $SYSROOT_PREFIX/usr/lib/$PKG_LIBNAME
   echo "set($PKG_LIBVAR $SYSROOT_PREFIX/usr/lib/$PKG_LIBNAME)" > $SYSROOT_PREFIX/usr/lib/cmake/$PKG_NAME/$PKG_NAME-config.cmake
-}
-
+}
```
Comment (MilhouseVH) [inline packages/emulation/libretro-picodrive/package.mk]: And again

**Comment by MilhouseVH** [comment]
This all looks very good so far - many thanks!

**Comment by CvH** [comment]
fixed

**Comment by MilhouseVH** [comment]
Many thanks @CvH!

**Pull request merged**